test/formatter_test.exs

```diff
@@ -317,6 +317,18 @@ defmodule FormatterTest do
       assert JUnitFormatter.get_report_file_path() ==
                "#{Mix.Project.app_path()}/junit_formatter-report_file_test.xml"
     end
+
+    test "create directory at init" do
```

Comment:
Can we just create another test that will run init with the dir already created?